nipype/algorithms/tests/test_compcor.py

```diff
@@ -142,7 +142,8 @@ def run_cc(self, ccinterface, expected_components, expected_header='CompCor'):
             components_data = [line.split('\t') for line in components_file]
 
             header = components_data.pop(0) # the first item will be '#', we can throw it out
-            expected_header = [expected_header + str(i) for i in range(expected_n_components)]
+            expected_header = [expected_header + '{:02d}'.format(i) for i in
+                               range(expected_n_components)]
```

Comment:
It's ok to change those names now, but after this will be released we need to be more careful since it's technically an API change.